### DISCUSSION TITLE
export feature

### DISCUSSION THREAD

**Discussion by rizaldi655** [Power-Components/livewire-powergrid#1336] [Q&A] [answered]
when i try export all column showing this ,
Class "OpenSpout\Writer\XLSX\Options" not found

**Comment by abhaypithadiya** [comment, marked as answer]
Hey! You need to install the package to resolve this issue.

[OpenSpout Installation](https://github.com/openspout/openspout/blob/4.x/docs/index.md.)

Hope this helps!
